cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastInstance.java

```diff
@@ -0,0 +1,137 @@
+package cwms.cda.data.dto.forecast;
+
+import cwms.cda.api.errors.FieldException;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.xml.bind.annotation.*;
+import java.time.Instant;
+import java.util.Map;
+
+@XmlRootElement(name = "forecast-instance")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ForecastInstance {
+
+  @Schema(description = "Forecast Instance ID")
+  @XmlElement(name = "instance-id")
+  String instanceId;

```

Comment:
There is no id for the instance, the uniqueness constraint  s the spec + issue-date-time